lib/src/main/java/com/wherobots/db/jdbc/WherobotsJdbcDriver.java

```diff
@@ -47,6 +48,18 @@ public class WherobotsJdbcDriver implements Driver {
     public static final Runtime DEFAULT_RUNTIME = Runtime.SEDONA;
     public static final Region DEFAULT_REGION = Region.AWS_US_WEST_2;
 
+    public Map<String, String> getUserAgentHeader() {
```

Comment:
A test would be nice to cover this

Reply:
Weird, I remember I added one, somehow not committed, let me add it